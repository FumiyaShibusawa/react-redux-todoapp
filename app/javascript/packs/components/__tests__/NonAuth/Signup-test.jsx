import * as React from 'react';
import SignupWithErrorBoundary from '../../NonAuth/Signup';
import renderer from 'react-test-renderer';

test('it renders correctly', () => {
  const tree = renderer.create(
    <SignupWithErrorBoundary />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
