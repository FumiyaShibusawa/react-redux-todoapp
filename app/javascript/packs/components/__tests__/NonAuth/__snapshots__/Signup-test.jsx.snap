// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly 1`] = `
<div
  className="signup-form"
>
  <h1>
    Register
  </h1>
  <form
    action="/users"
    charSet="UTF-8"
    method="POST"
  >
    <input
      name="authenticity_token"
      type="hidden"
    />
    <input
      autoComplete="username"
      name="user[name]"
      onChange={[Function]}
      placeholder="Enter your name"
      type="text"
    />
    <input
      autoComplete="email"
      name="user[email]"
      placeholder="Enter your email"
      type="text"
    />
    <input
      autoComplete="current-password"
      name="user[password]"
      placeholder="Enter your password"
      type="password"
    />
    <button
      type="submit"
    >
      Register
    </button>
  </form>
  <div
    className="goto-login"
    data-modal="popup"
  >
    Already have your account?
  </div>
</div>
`;
